repo/packages/M/mongodb/0/resource.json

```diff
@@ -0,0 +1,17 @@
+{
+  "images": {
+    "icon-small": "http://www.file-extensions.org/imgs/app-icon/48/8854/mongodb-icon.png",
+    "icon-medium": "http://www.sudops.com/wp-content/uploads/2014/02/mongodb-96x96.png",
+    "icon-large": "http://1.bp.blogspot.com/-FagQLEI_Zhc/VnAzE9L8e3I/AAAAAAAAAFk/DWPqvL0ZXo8/s320/mongodb.jpg",
```

Comment:
Understood and I'd love to. Unfortunately I don't have access to that server or know how to upload images there. The current version for this package works the same.